gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest_asyncio.py.j2

```diff
@@ -22,6 +22,8 @@ from google.api_core import exceptions as core_exceptions
 from google.api_core import gapic_v1
 from google.api_core import retry_async as retries
 from google.api_core import rest_helpers
+{# TODO(https://github.com/googleapis/gapic-generator-python/issues/2137): raise an import error if an older version of google.api.core is installed. #}
+from google.api_core import rest_streaming_async  # type: ignore
```

Comment:
There should also be a TODO comment for the `# type: ignore` here (unless that TODO comment if for the `# type: ignore`

Reply:
I think this will be removed when we address https://github.com/googleapis/gapic-generator-python/issues/2187 before releasing the feature.